core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/error/ErrorManagerImpl.java

```diff
@@ -23,145 +23,51 @@
  */
 package org.silverpeas.core.workflow.engine.error;
 
-import java.util.Vector;
-
 import org.silverpeas.core.workflow.api.ErrorManager;
-import org.silverpeas.core.workflow.api.WorkflowException;
 import org.silverpeas.core.workflow.api.error.WorkflowError;
-import org.silverpeas.core.workflow.api.instance.HistoryStep;
-import org.silverpeas.core.workflow.engine.jdo.WorkflowJDOManager;
-import org.silverpeas.core.silvertrace.SilverTrace;
-
 import org.silverpeas.core.workflow.api.event.GenericEvent;
+import org.silverpeas.core.workflow.api.instance.HistoryStep;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.workflow.api.instance.HistoryStep'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)